WordPress/src/main/java/org/wordpress/android/ui/reader/viewmodels/ReaderPostListViewModel.kt

```diff
@@ -131,7 +144,7 @@ class ReaderPostListViewModel @Inject constructor(
         newsTracker.trackNewsCardExtendedInfoRequested(READER, item.version)
     }
 
-    fun loadSubFilters() {
+    fun onLoadSubFilters() {
```

Comment:
It seems the method actually loads the subfilters, we might want to consider renaming it back to `loadSubFilters`.

Reply:
Changed here 89dab3793dbf1c6bdaa270ee12d007a3532b8f46